src/components/ui/upgrade-dialog/index.tsx

```
@@ -8,107 +12,259 @@ import {
+          <div className="grid gap-4 md:grid-cols-3">
+            {plans.map((plan) => {
+              const isSelected = selectedPlan === plan.id;
+              const isCurrent = currentPlan === plan.id;
+              const limit = formatLimit(getLimitForPlan(plan, limitType));
+              const canCheckout = Boolean(plan.pricing);
+
+              return (
+                <div
+                  key={plan.id}
+                  role="button"
+                  aria-pressed={isSelected}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)